docs/network-iot/run-an-lns/configure-chirpstack.mdx

```diff
@@ -0,0 +1,88 @@
+
+- Helium Network Organizationally Unique Identifier (OUI)
+- Helium LoRaWAN devAddr slab
+
+<!-- TODO: what is config service? -->
+
+### Set Up Chirpstack Docker
+
+1. git clone https://github.com/chirpstack/chirpstack-docker.git
+2. Open configuration/chirpstack/chirpstack.toml
+   1. Under `[network]` Change `net_id` to `0x00003C`
+   2. Under `[network]`, add `dev_addr_prefixes` with the devAddr slab assigned to you. First run
+      `./main org get --OUI <your OUI>` to get your OUI's DevAddr constraints. Then run
+      `./main subnet-mask <your OUI's starting DevAddr> <your OUI's ending DevAddr>` to find your
+      subnets. In the chirpstack toml file put in `dev_addr_prefixes` = [`<your subnets>`], which is
+      formatted with the following rule: `starting devAddr`/`32-log2(devAddr slab size)`. Let's say
+      you requested 8 devAddr slab and the starting devAddr is `00000000`, then you would input
+      `dev_addr_prefixes` = ["00000000/29"]
+   3. Under `[gateway]`, add a line `allow_unknown_gateways=true`
+
+### Set Up Supported LoRaWAN Region
+
+You need to deploy a gateway bridge for each LoRaWAN region you plan on supporting on Chirpstack. By
+default, Chirpstack only supports EU868. This tutorial will show you how to add support for US915,
+and you can replicate the same process for other LoRaWAN regions.
+
+Open docker-compose.yml, you will find a container called `chirpstack-gateway-bridge-eu868`. Add the
+following.
+
+```
+chirpstack-gateway-bridge-us915:
+    image: chirpstack/chirpstack-gateway-bridge:4
+    restart: unless-stopped
+    ports:
+      - 1701:1700/udp
+    volumes:
+      - /helium/configuration/chirpstack-gateway-bridge-us915:/etc/chirpstack-gateway-bridge
+    depends_on:
+      - mosquitto
+    logging: *logs
+```
+
+Now head over to `configuration/chirpstack-gateway-bridge` and create a directory called
+`chirpstack-gateway-bridge-us915`. In that directory, create a file called
+`chirpstack-gateway-bridge.toml`. In that file, put down the following.
+
+```
+# See https://www.chirpstack.io/gateway-bridge/install/config/ for a full
+# configuration example and documentation.
+
+[integration.mqtt.auth.generic]
+servers=["tcp://mosquitto:1883"]
+username=""
+password=""
+
+[integration.mqtt]
+event_topic_template="us915_1/gateway/{{ .GatewayID }}/event/{{ .EventType }}"
+state_topic_template="us915_1/gateway/{{ .GatewayID }}/state/{{ .StateType }}"
+command_topic_template="us915_1/gateway/{{ .GatewayID }}/command/#"
+```
+
+Make sure you have port 1700 and 1701 both open on your server.
+
+### Run Your Chirpstack
+
+Run `docker-compose up` and you should be able to log in with the default credential:
+
+- username = admin
+- password = admin
+
+Now that you have your LNS up and running, the last step is to configure routing rules, so that the
+network knows which LNS to send device packets too. Learn how to do that
+[here](configure-routing-rules).
```

Comment:
Maybe add a warning to not run this in production / exposed to the internet like this

Please note: this configuration is not production ready. It is a good starting point to experiment; but before  you run this on a server that is exposed to the internet, please know what the security implications are.